database/services/save.go

```diff
@@ -5,18 +5,17 @@ import (
 	"github.com/mesg-foundation/core/service"
 )
 
-// Save a service in the database and return the hash or the error if something wrong happened
+// Save stores a service in the database and returns a hash or an error.
 func Save(service *service.Service) (hash string, err error) {
 	bytes, err := proto.Marshal(service)
 	if err != nil {
-		return
+		return "", err
 	}
 	db, err := open()
 	defer close()
 	if err != nil {
-		return
+		return "", err
 	}
 	hash = service.Hash()
```

Comment:
to stay consistent with the result of the refactoring, I would not use the named output and assign a new variable here

Reply:
...but the whole package consists of named outputs.
Should we refactor service package to remove named outputs?

Reply:
hum no not really, let's do things step by step, i'm ok with that anyway